openedx/core/djangoapps/user_authn/views/utils.py

```diff
@@ -108,3 +113,60 @@ def get_mfe_context(request, redirect_to, tpa_hint=None):
         'countryCode': country_code,
     })
     return context
+
+
+def generate_username(username_initials):
```

Comment:
Its the learner's name initials
```suggestion
def generate_username(learner_name_initials):
```